src/main/resources/hibernate/custom-index-settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "number_of_shards": "4",
   "number_of_replicas": "0",
-  "max_ngram_diff": "20",
+  "max_ngram_diff": "29",
```

Comment:
@markquintontulloch there is also a local-index-setting file. Those will need to be changed also otherwise it wont work when deving locally.

Reply:
@mluypaert this also has an impact on index size, might want to keep an eye on it.